sdk/spring/scripts/ci_update_versions.py

```diff
@@ -0,0 +1,190 @@
+# Python version 3.4 or higher is required to run this script.
+#
+# Change the external_dependencies in sdk/spring for test compatibility of spring-boot version .
+#
+# How to use:
+# 1. Make sure file(`.\sdk\spring\spring_boot_SPRING_BOOT_VERSION_managed_external_dependencies.txt`) exist. If it doesn't exist, please run
+#    `.\sdk\spring\scripts\get_spring_boot_managed_external_dependencies.py` to create that file.
+# 2. Change `SPRING_BOOT_VERSION` in this script manually.
+# 3. Then the ci will automatically run command `python .\sdk\spring\scripts\ci_update_versions.py --ut external_dependency`.
+#
+# The script must be run at the root of azure-sdk-for-java.
+
+import argparse
+from datetime import timedelta
+import os
+import re
+import sys
+import time
+import traceback
+from ci_utils import CodeModule
+from ci_utils import external_dependency_version_regex
+from ci_utils import external_dependency_include_regex
+from ci_utils import UpdateType
+from ci_utils import include_update_marker
+from ci_utils import version_regex_str_no_anchor
+from ci_utils import version_update_start_marker
+from ci_utils import version_update_end_marker
+from ci_utils import version_update_marker
+
+SPRING_BOOT_VERSION = '2.6.1'
+
+SPRING_BOOT_MANAGED_EXTERNAL_DEPENDENCIES_FILE_NAME = 'sdk/spring/scripts/spring_boot_{}_managed_external_dependencies.txt'.format(SPRING_BOOT_VERSION)
+
+def update_versions(update_type, version_map, ext_dep_map, target_file):
+    #list for new pom
+    newlines = []
+    repl_open, repl_line, file_changed, is_include = False, False, False, False
+    print('processing: ' + target_file)
+    try:
+        with open(target_file, encoding='utf-8') as f:
```

Comment:
How about use replace instead of create new file?
Just like what we did in `sync_external_dependencies.py`.

```
    with in_place.InPlace(target_file) as file:
        for line in file:
            line = line.strip()
            if line.startswith('#') or not line:
                file.write(line)
```